docs/src/10-font-line-height.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Meta, Typeset } from '@storybook/addon-docs/blocks';
+import { TokenTable } from './lib/TokenTable';
+import { font } from '@wmde/wikit-tokens';
+import styles from './styles/line-height.css';
+
+<Meta title="Design Tokens|Font line-height" />
+
+## line-height
+
+<TokenTable
+    tokens={font[ 'line-height' ]}
+    valueCell={( value ) => (
+        <div>
+            <pre>{ value }</pre>
+            <div className='lineHeight' style={{ lineHeight: value }}>demo</div>
```

Comment:
```suggestion
            <div className='lineHeight' style={{ lineHeight: value }}>
            The quick brown fox jumps over the lazy dog
            </div>
```
I think the visuals can benefit from a longer sentence. It makes the line height visible between lines as well.
Like here https://www.lightningdesignsystem.com/design-tokens/#category-line-height

Reply:
Hmm. I guess for me it became less clear, or maybe I misunderstood.

Reply:
Ok, we can ask UX first, before we merge this one.
I think line-height is most important in paragraphs.